dev/wlp-jakartaee-transform/rules/jakarta-direct.properties

```diff
@@ -40,3 +40,13 @@ javax.servlet.async.mapping=jakarta.servlet.async.mapping
 /javax/servlet/resources/j2ee_1_4.xsd=/jakarta/servlet/resources/j2ee_1_4.xsd
 /javax/servlet/resources/xml.xsd=/jakarta/servlet/resources/xml.xsd
 /javax/servlet/resources/j2ee_web_services_client_1_1.xsd=/jakarta/servlet/resources/j2ee_web_services_client_1_1.xsd
+
+# JSP
+javax.naming.InitialContext=jakarta.naming.InitialContext
+javax.faces.passthrough.Element=jakarta.faces.passthrough.Element
+javax.faces.request.charset=jakarta.faces.request.charset
+javax.faces.visit.SKIP_ITERATION=jakarta.faces.visit.SKIP_ITERATION
+javax.faces.resource.Script=jakarta.faces.resource.Script
+javax.servlet.error.message=jakarta.servlet.error.message
+javax.servlet.error.exception=jakarta.servlet.error.exception
+javax.servlet.jsp.jstl.fmt.locale=jakarta.servlet.jsp.jstl.fmt.locale
```

Comment:
Can you organize this a bit better by technology? I assume these are for Faces and not JSP, however there are some servlet updates and we have more that we need to make here see https://github.com/OpenLiberty/open-liberty/issues/13971 which I'm currently working on.

Reply:
Looks like a rebase I performed wasn't properly done.  I can fix this up.

Reply:
Actually, these might not be necessary all for this this PR. However, I think they should be looked at later on. Particularly those JSP attributes. I've create https://github.com/OpenLiberty/open-liberty/issues/14010.

Reply:
Scratch that. Existing issue already exists.